challenge.rb

```diff
@@ -1,26 +1,27 @@
 
-def capitalize_each_string(input)
+def capitalize_each_string(*animal)
   #implement your solution here
+  animal.each { |a| puts a.capitalize }
```

Comment:
> Since the map method presumably changed the array to all caps, why didn't we need to downcase the array prior to making the other tests pass?
The `map` method returns a copy of the original Array with the modifications. The original Array is not changed, so there is no need to revert the change. Note that `map!` will edit the Array in place, and that `collect` is just an alias for `map`.

> What's the difference between the find and find_all methods? I was surprised that the find method did not return True.